doc/conf.py

```diff
@@ -254,9 +239,10 @@
 # display the source code for Plot directive
 plot_include_source = True
 
+
 def setup(app):
     app.add_stylesheet("pysal-styles.css")
```

Comment:
Did you want to switch this according to #134?

Reply:
Another good catch. I thought I had changed this. Will do now.